modules/programs/jetbrains-remote.nix

```
@@ -31,7 +31,7 @@ in {
         "${ide}/bin/${ide.meta.mainProgram}-remote-dev-server registerBackendLocationForGateway || true";
       lines = map mkLine cfg.ides;
       linesStr = ''
-        rm $HOME/.cache/JetBrains/RemoteDev/userProvidedDist/_nix_store*
+        rm $HOME/.cache/JetBrains/RemoteDev/userProvidedDist/_nix_store* || true
```

Wondering if we should do `rm -f` instead, it's specifically for that purpose and might allow other errors to show through.

---

`rm -f` returns non-zero exit code if it fails to delete a file. That was my initial idea. We can use it to silence an error that comes from rm command though, but otherwise activation script will fail without `|| true`

---

Alright, fair enough. Thanks